snipssonos/use_cases/hotword/restore_volume.py

```diff
@@ -0,0 +1,28 @@
+from snipssonos.entities.device import Device
+from snipssonos.shared.use_case import UseCase
+from snipssonos.shared.response_object import ResponseSuccess, ResponseFailure
+from snipssonos.exceptions import NoReachableDeviceException
+from snipssonos.use_cases.request_objects import VolumeSetRequestObject
+from snipssonos.use_cases.volume.set import VolumeSetUseCase
+
+
+class HotwordRestoreVolumeUseCase(UseCase):
+
+    def __init__(self, device_discovery_service, device_transport_control_service, state_persistence_service):
+        self.device_discovery_service = device_discovery_service
+        self.device_transport_control_service = device_transport_control_service
+        self.state_persistence_service = state_persistence_service
+
+    def process_request(self, request_object):
+        devices = self.state_persistence_service.get_all(Device)
+        if len(devices):
+            FIRST_DEVICE_VOLUME = devices[0].volume
```

Comment:
As you already did the effort to actually save the state of all the devices, it takes nothing to reset the volume to all of them with a loop here. It is not a big effort and already tackles a future feature.

Reply:
Also `FIRST_DEVICE_VOLUME ` should be lowercased, it is not a global.